resources/application-connector/charts/application-operator/values.yaml

```diff
@@ -13,7 +13,7 @@ controller:
       cpu: 100m
       memory: 256Mi
     requests:
-      cpu: 100m
+      cpu: 10m
```

Comment:
This one should not be changed. This component may get resource hungry when there are multiple Service Instances, we already sometimes have problems with the limit here.